modules/nf-core/bascule/templates/main_script.R

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env Rscript
+
+parse_args = function(x) {
```

Comment:
This is very different from other parse_args R functions that I've seen in nf-core, see for example https://github.com/nf-core/modules/blob/efc7b2793391bb02bef37fb55d52e41b0b8b8839/modules/nf-core/gprofiler2/gost/templates/gprofiler2_gost.R#L41. 
I'm assuming that the standard one didn't work. Are the arguments to bascule very different from other R using modules?